airgun/views/common.py

```diff
@@ -56,6 +62,81 @@ class listremove(SatSecondaryTab):
         './/nav[@class="ng-scope"]/following-sibling::div')
 
 
+class LCESelectorGroup(ParametrizedView):
+    """Group of :class:`airgun.widgets.LCESelector`, typically present on page
+    for selecting desired lifecycle environment.
+
+    Usage::
+
+        lce = View.nested(LCESelectorGroup)
+
+        #or
+
+        @View.nested
+        class lce(LCESelectorGroup):
+            pass
+    """
+    ROOT = ".//div[@path-selector='environments']"
+
+    PARAMETERS = ('lce_name',)
+
+    LAST_ENV = ".//div[contains(@class, 'path-selector')]/ul/li[last()]"
+    lce = LCESelector(
+        locator=ParametrizedLocator(
+            "./div[contains(@class, 'path-selector')]/ul"
+            "[li[normalize-space(.)='{lce_name}']]")
+    )
+
+    @classmethod
+    def all(cls, browser):
+        """Helper method which returns list of tuples with all LCESelector
+        names (last available environment is used as a name). It's required for
+        :meth:`read` to work properly.
+        """
+        return [(element.text,) for element in browser.elements(cls.LAST_ENV)]
+
+    def fill(self, values=None):
+        """Shortcut to pass the value to included ``lce``
+        :class:`airgun.widgets.LCESelector` widget. Usage remains the same as
+        :class:`airgun.widgets.LCESelector` and
+        :class:`widgetastic.widget.ParametrizedView` required param is filled
+        automatically from passed lifecycle environment's name.
+
+        Example::
+
+            my_view.lce.fill({'PROD': True})
+
+        Value ``True`` or ``False`` means to set corresponding checkbox value
+        to the last checkbox available in widget (last lifecycle environment).
+        If you want to select different lifecycle environment within the same
+        route - pass its name as a value instead::
+
+            my_view.lce.fill({'PROD': 'Library'})
```

Comment:
does it mean that `my_view.lce.fill({'PROD': 'Library'})` can only set value to True for specific LCE?